src/Database/Schema/Column.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace TCG\Voyager\Database\Schema;
+
+use Doctrine\DBAL\Schema\Column as DoctrineColumn;
+use Doctrine\DBAL\Types\Type;
+use Doctrine\DBAL\Schema\Comparator;
+
+class Column
```

Comment:
What was the reason for throwing out what you did in #623? Specifically, extending DoctrineColumn, as opposed to how you're doing it here by passing it into the constructor.